setup.py

```diff
@@ -43,6 +43,7 @@ def get_setup_version(reponame):
     license='BSD',
     url='http://pyviz.org',
     packages = ["pyviz_comms"],
+    package_data={'pyviz_comms': ['.version']},
```

Comment:
Shouldn't it be sufficient to add it to the MANIFEST?

Reply:
No! I checked multiple times. Without it, the ``.version`` file doesn't end up in the archive built with ``conda build .``. I don't get it either but it is needed...

Reply:
Okay, just noticed the other packages using autover also have it.

Reply:
@jlstevens/@philippjfr, I made a note about this:
https://github.com/pyviz/pyct/wiki/Packaging-tips

E.g. https://github.com/pyviz/nbsite/blob/master/setup.py

Note you need to be worried not only what ends up in a package, but also what gets installed.

It'll be great when we do it uniformly, but right now we're using package_data in many projects (for the examples, usually).